Menu/Menu.js

```diff
@@ -1,12 +1,7 @@
 
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
-  if (menu.style.display === "none") {
-    menu.style.display = "block";
-  }
-  else {
-    menu.style.display = "none";
-  }
+    menu.classList.toggle("menu--open");
 }
```

Comment:
here you could just toggle the class for toggleMenu() To work and check if menu.classList.contains that specific class in order to apply different effects to it